linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminHandler.scala

```diff
@@ -59,11 +60,18 @@ class AdminHandler(navItems: Seq[NavItem]) extends HtmlView {
     content: String,
     tailContent: String = "",
     csses: Seq[String] = Nil,
-    navbar: String = navBar
+    navbar: String = "",
+    navHighlight: String = ""
   ): String = {
     val cssesHtml = csses.map { css =>
       s"""<link type="text/css" href="files/css/$css" rel="stylesheet"/>"""
     }.mkString("\n")
+
+    val navbarWithHighlight = navbar match {
+      case "" => navBar(navHighlight)
+      case _ => navbar
+    }
+
```

Comment:
Hmm, this is a confusing change to how `html` works w/o navbar or navHighlight specified. `html` used to display the navbar if none was specified. In this version `html` displays no navbar if navbar & navHighlight are unspecified.

I suggest we change the method signature to make this logic easier to follow, something like:

```
customNavbar: Option[String] = None,
navHighlight: Option[String] = None
): String = {
...
val navbarWithHighlight = (customNavbar, navHighlight) match {
  case (Some(custom), _) => custom
  case (None, None) => navBar()
  case (None, Some(highlight)) => navBar(highlight)
}
```

or just get rid of the navbar arg entirely, as it's not used anywhere that I can see